lib/prehog/proto/prehog/v1alpha/teleport.proto

```diff
@@ -107,6 +126,8 @@ message SubmitEventRequest {
     UserLoginEvent user_login = 3;
     SSOCreateEvent sso_create = 4;
     ResourceCreateEvent resource_create = 5;
+
+    // REMOVE IN V14: Use session_start_v2 instead
```

Comment:
We can remove this way sooner than v14, we can remove it as soon as the last tenant on cloud uses the new event.

Reply:
I figure it's more of a "remove by v14 at the latest" so we can remove it earlier if we'd like... in any case it's part of the prehog proto so it's hard to change from this end :(